### PR TITLE
Update footer button transitions to match other website elements

### PR DESCRIPTION
**Pull request by maticbasle**
This PR updates the footer buttons to include transitions to match other website elements such as the buttons on the index page.

### CODE REVIEW

File: site/layouts/partials/footer.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
     <div class="container-fluid">
         <div class="footer__copyright">© Mattermost, Inc. All Rights Reserved.</div>
         <div class="footer__icons">
-            <a href="{{ .Site.Params.footer.facebook }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-facebook"></i></a>
-            <a href="{{ .Site.Params.footer.twitter }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a>
-            <a href="{{ .Site.Params.footer.youtube }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-youtube"></i></a>
+            <a href="{{ .Site.Params.footer.facebook }}" target="_blank" rel="noopener noreferrer" class="btn-primary" style="transition: background-color .15s ease-in-out"><i class="fa fa-facebook"></i></a>
```
Comment (hanzei) [inline site/layouts/partials/footer.html:5]: Good catch :+1: 

What do you think about moving the `transition: background-color .15s ease-in-out` into `style.css`?

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR :+1: 

One suggestion on the code organisation

File: site/layouts/partials/footer.html (hunk 1 above)
Comment (maticbasle) [reply site/layouts/partials/footer.html:5]: To apply the transition to all primary buttons or create a new class?

Comment (hanzei) [reply site/layouts/partials/footer.html:5]: Can we add it to the `footer__icons` class?

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by abhijit-singh — APPROVED** [review]

**Pull request merged**